cmd/protoc-gen-go-helpers/main.go

```diff
@@ -59,7 +59,7 @@ type (
 const (
 	msgImport  = "google.golang.org/protobuf/proto"
 	enumImport = "fmt"
-	headerTmpl = `
+	headerTmpl = `// Code generated by protoc-gen-go-helpers. DO NOT EDIT.
```

Comment:
I'd add a blank line in between just so it doesn't look like a doc comment on the package